test/test_0000_imports.py

```diff
@@ -29,8 +29,12 @@ def test_0000_imports() -> None:
     from al_bench.model import AbstractModelHandler  # noqa F401
     from al_bench.model import Logger  # noqa F401
     from al_bench.model import ModelStep  # noqa F401
-    from al_bench.model import PyTorchModelHandler  # noqa F401
-    from al_bench.model import TensorFlowModelHandler  # noqa F401
+    from al_bench.model import NonBayesianPyTorchModelHandler  # noqa F401
```

Comment:
Maybe we could just call this the PytorchModelHandler?

Reply:
Just pushed that fix.